.Rbuildignore

```diff
@@ -1,3 +1,4 @@
 ^rv\.git\.pkgA\.Rproj$
 ^\.Rproj\.user$
 ^LICENSE\.md$
+rproject.toml
```

Comment:
also need to add the `rv` directory + the new `.Rprofile` so it doesn't bundle those in!